website/docs/docs/ai/managing.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Managing PactFlow AI
+sidebar_label: Managing PactFlow AI
+---
+
+## Usage Limits
+
+Access to the feature is metered based on the number of tests generated in a 30-day rolling window. Credits are allocated as follows:
```

Comment:
This is still true as of writing this, by Prateek's PR will soon change this to be calendar-month based.

```suggestion
Access to the feature is metered based on the number of tests generated per calendar month and reset on the 1st of each month. Credits are allocated as follows:
```